README.md

```diff
@@ -52,7 +52,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link]()
+- [Live Demo Link coming soon]()
 
 <!-- GETTING STARTED -->
 
```

Comment:
- [ ] Kindly remove the `robot.txt` file as it won't be used for subsequent project.

  ![qoms](https://user-images.githubusercontent.com/36542986/220579786-0a9844b1-3b76-43e1-b857-9f67d9a4bba6.jpg)